test/e2e/apimachinery/aggregator.go

```diff
@@ -107,7 +107,7 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 
 	namespace := f.Namespace.Name
 	context := setupServerCert(namespace, "sample-api")
-	if framework.ProviderIs("gke") {
+	if framework.ProviderIs("gke") || framework.ProviderIs("gce") {
```

Comment:
what's going on here that is provider specific? ie: why couldn't this run on aws